content/2016-12-26_komunitni-python-stanek-na-linuxdays.md

```diff
@@ -0,0 +1,90 @@
+Title: Komunitní Python stánek na LinuxDays
+Date: 2016-12-26 13:28:00
+Modified: 2016-12-26 13:28:00
+Author: Veronika Gabrielová
+Twitter: lspdv
+Gravatar: gabrielova.veronika@gmail.com
+About: Veronička je velmi aktivní v české [Python](http://python.cz/) komunitě. Motá se hodně kolem [PyLadies](http://pyladies.cz/), účastnicím po kurzech pomáhá najít vysněnou práci a právě buduje rozjezd i dalších navazovacích kurzů a srazů pro všechny zájemce o Python, kteří jsou v začátcích. Také již pracuje jako programátorka. Nemá ráda datle. Má ráda vše ostatní kromě datlí. Nejvíc Žufánka a Pho. Nejčastěji ji potkáš v Praze na Pyvo srazech či na PyWorkingu a PyLadies.
+
+### Organizátoři LinuxDays nás oslovili, zdali chceme využít možnosti prezentovat se na známé akci LinuxDays. 
+
+```
```

Comment:
Treba na GH je to pozadi sedy, tak jsem myslela, ze se tak vseobecne meni pozadi. Ze to je jen na kod, slysim prvne. 

Pokud chcete, aby lidi psali clanky, musite pocitat s tim, ze nevedi vsechno o MD. Proto se jim to muze trochu zesnadnit v dokumentaci.